### PR TITLE
Address Issue 84: Align groupings with the RFCs

### PR DESCRIPTION
**Pull request by mjethanandani**
This PR address issue #84

### CODE REVIEW

File: src/yang/ietf-qos-action.yang [hunk 1]
```diff
@@ -541,22 +544,17 @@ module ietf-qos-action {
 
       leaf burst-value {
         type uint64;
+        units "bytes-per-second";
```
Comment (aseemavi) [inline src/yang/ietf-qos-action.yang:547]: burst unit in "bytes" and milli/microsecond.. Why not keep a unit just like rate above

Comment (mahesh-arrcus) [reply src/yang/ietf-qos-action.yang:547]: Because that is what the RFC suggests

Comment (aseemavi) [reply src/yang/ietf-qos-action.yang:547]: you mean the Yang RFC? separate unit gives flexibility to define bust size in bytes and time.

Comment (mjethanandani) [reply src/yang/ietf-qos-action.yang:547]: No. RFC 2697.

Besides you have a large type defined to be able to define different burst sizes.

Comment (aseemavi) [reply src/yang/ietf-qos-action.yang:547]: max-rate is for queues.. check for diffServMaxRateThreshold in rfc 3289.. not sure what you mean by "large type"

File: src/yang/ietf-qos-action.yang [hunk 2]
```diff
@@ -541,22 +544,17 @@ module ietf-qos-action {
 
       leaf burst-value {
         type uint64;
+        units "bytes";
```
Comment (aseemavi) [inline src/yang/ietf-qos-action.yang:547]: since this is for Queues, let us retain the units. For Meters, it is fine since there is RFC reference.

**Review by aseemavi — APPROVED** [review]

**Pull request merged**